metadata_mapper/validate_mapping.py

```diff
@@ -207,6 +205,22 @@ def validate_mapped_page(rikolti_records, solr_records, query):
                     )
 
         query['start'] = query['start'] + 100
+        if query['start'] < num_solr_records:
+            collections_search = solr(**query)
+            collections_search = collections_search.get(
+                'response', {'docs': None}).get('docs', [])
+            solr_records.update(
+                {r['harvest_id_s']: r for r in collections_search})
+        else:
```

Comment:
Add handling for cases when Rikolti has new records that don't appear in Solr

Reply:
Makes sense. Wow yeah, the (necessary) logic for getting pages from solr and comparing to pages of rikolti records is convoluted...